transport/controller.cc

```diff
@@ -141,6 +141,10 @@ future<> controller::start_listening_on_tcp_sockets(sharded<cql_server>& cserver
 }
 
 future<> controller::start_listening_on_maintenance_socket(sharded<cql_server>& cserver) {
+    auto unix_domain_socket_permissions =
```

Comment:
nit: best to declare stuff close to where it's being used, perhaps consider moving this variable closer to listen_on_all_shards call